packages/config/config/devices/0x0344/he-tps05.json

```
@@ -0,0 +1,970 @@
+					"value": 0
+				},
+				{
+					"label": "Relay inverts state according to the relay state",
+					"value": 1
+				},
+				{
+					"label": "Relay inverts state according to the button backlight state",
+					"value": 2
+				},
+				{
+					"label": "Turn on",
+					"value": 3
+				},
+				{
+					"label": "Turn off",
+					"value": 4
+				},
+				{
+					"label": "Turn on, then off after timer",
+					"value": 5
+				},
+				{
+					"label": "Turn off, then on after timer",
+					"value": 6
+				}
+			]
+		},
+		"53": {
+			"label": "Button 3: Click Control Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Relay inverts state according to the relay state",
+					"value": 1
+				},
+				{
+					"label": "Relay inverts state according to the button backlight state",
+					"value": 2
+				},
+				{
+					"label": "Turn on",
+					"value": 3
+				},
+				{
+					"label": "Turn off",
+					"value": 4
+				},
+				{
+					"label": "Turn on, then off after timer",
+					"value": 5
+				},
+				{
+					"label": "Turn off, then on after timer",
+					"value": 6
+				}
+			]
+		},
+		"54": {
+			"label": "Button 4: Click Control Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Relay inverts state according to the relay state",
+					"value": 1
+				},
+				{
+					"label": "Relay inverts state according to the button backlight state",
+					"value": 2
+				},
+				{
+					"label": "Turn on",
+					"value": 3
+				},
+				{
+					"label": "Turn off",
+					"value": 4
+				},
+				{
+					"label": "Turn on, then off after timer",
+					"value": 5
+				},
+				{
+					"label": "Turn off, then on after timer",
+					"value": 6
+				}
+			]
+		},
+		"55": {
+			"label": "Button 5: Click Control Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Relay inverts state according to the relay state",
+					"value": 1
+				},
+				{
+					"label": "Relay inverts state according to the button backlight state",
+					"value": 2
+				},
+				{
+					"label": "Turn on",
+					"value": 3
+				},
+				{
+					"label": "Turn off",
+					"value": 4
+				},
+				{
+					"label": "Turn on, then off after timer",
+					"value": 5
+				},
+				{
+					"label": "Turn off, then on after timer",
+					"value": 6
+				}
+			]
+		},
+		"61": {
+			"label": "Relay 1: Button number source",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Controlled by gateway or associated device",
```

```suggestion
					"label": "Gateway or associated device",
```

Thinking of, doesn't this just mean the relay is controlled via Z-Wave?